package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "powershell-preview",
-  "displayName": "PowerShell Preview",
-  "version": "2020.1.0",
-  "preview": true,
+  "name": "powershell",
```

Comment:
I thought we were gonna leave this as preview and then spin out a release branch with the name change? Otherwise we could run into the accidental "release preview to stable" issue that we ran into a while back.